src/WebJobs.Extensions.OpenAI/Assistants/AssistantQueryAttribute.cs

```diff
@@ -26,4 +26,14 @@ public AssistantQueryAttribute(string id)
     /// </summary>
     [AutoResolve]
     public string TimestampUtc { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Configuration section name for the table settings for chat storage.
+    /// </summary>
+    public string? ChatStorageConnectionSetting { get; set; }
```

Comment:
did we want to add AutoResolve here as well? I see the other attributes have it so wanted to check

Reply:
we can use it for CollectionName but not for ChatStorageConnectionSetting as we need to pass the setting name and dont want to auto resolve it.

I don't see AutoResolve attribute being used for Service Bus bindings.

@cgillum - any suggestions here?

Reply:
I think it makes sense to stick with precedent. But I suspect part of the reason why connections aren't auto-resolve properties is because they already reference app settings, so making this into an auto-resolve property may be redundant.